x-pack/plugins/synthetics/scripts/base_e2e.js

```diff
@@ -71,13 +71,23 @@ function executeSyntheticsRunner(dirPath) {
       }
     );
   } else if (runner) {
-    childProcess.execSync(
-      `node ../../../../scripts/${ftrScript} --config ${config} --kibana-install-dir '${kibanaInstallDir}'  --headless ${headless} --bail ${pauseOnError} --grep '${grep}'`,
-      {
-        cwd: dirPath,
-        stdio: 'inherit',
-      }
-    );
+    if (grep) {
+      childProcess.execSync(
+        `node ../../../../scripts/${ftrScript} --config ${config} --kibana-install-dir '${kibanaInstallDir}'  --headless ${headless} --bail ${pauseOnError} --grep '${grep}'`,
```

Comment:
Instead of doing the conditional repeating the command, perhaps in the command we can do `${grep ? '--grep ' + grep : ''}`

Reply:
yeah that would be clean.